File: py files/2a.py
# 2a: Write a program in Python that prints numbers from 1 to 10 using a for loop.

def print_numbers_for_loop():
    print("Numbers from 1 to 10:")
    for i in range(1, 11):
        print(i, end=" ")
    print()

if __name__ == "__main__":
    print_numbers_for_loop()

File: js files/2a.js
// 2a. Write a program in JS that prints numbers from 1 to 10 using a for loop.

function printNumbersForLoop() {
    console.log("Numbers from 1 to 10:");
    let output = "";
    
    // FIXED: Changed i < 10 to i <= 10 to include 10
    for (let i = 1; i <= 10; i++) {
        output += i + " ";
    }
    
    console.log(output);
}

printNumbersForLoop();